src/api/app/controllers/webui/driver_update_controller.rb

```diff
@@ -107,7 +107,7 @@ def save
   # TODO: select architecture of binary packages
   def binaries
     required_parameters :repository
-    @binaries = @package.build_result(params[:repository], ['binarylist', 'status']).each('//binary').map{|binary| binary['filename']}
+    @binaries = @package.build_result_old(params[:repository], ['binarylist', 'status']).each('//binary').map{|binary| binary['filename']}
```

Comment:
Trello card in fruit tree created: [get rid of build_result_old](https://trello.com/c/LYoHQLqv/214-get-rid-of-build-result-old-in-app-controllers-webui-driver-update-controller-rb)